src/Microsoft.Azure.Batch.SoftwareEntitlement.Client.Native/Microsoft.Azure.Batch.SoftwareEntitlement.Client.Native.vcxproj

```diff
@@ -23,7 +23,7 @@
     <ProjectGuid>{6ACCAD67-C207-4370-B39A-63651223A21C}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
     <RootNamespace>MicrosoftAzureBatchSoftwareEntitlementClientNative</RootNamespace>
-    <WindowsTargetPlatformVersion>10.0.14393.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.16299.0</WindowsTargetPlatformVersion>
```

Comment:
Is this needed or did VS update it to some recent version? (Wondering if this can affect customers trying to use this code on older versions...)

Reply:
Ok, should be fine as long as we stick to the core bits of the API 🙂